scripts/run_ulissigroup_co_cnn.py

```diff
@@ -44,7 +44,7 @@ def main_helper():
         "warmup_epochs": 1,
         "warmup_factor": 0.2,
     }
-    trainer = MDTrainer(
+    trainer = ForcesTrainer(
```

Comment:
This should probably use `SimpleTrainer`, and not `ForcesTrainer`. (Not sure why it was using MDTrainer earlier)

On a related note, we could probably rename `SimpleTrainer` to `EnergyTrainer` :)

Reply:
Yeah I was confused about this as well, will update.